nexus/types/src/deployment.rs

```diff
@@ -55,6 +55,13 @@ pub struct Policy {
     /// ranges specified by the IP pool for externally-visible control plane
     /// services (e.g., external DNS, Nexus, boundary NTP)
     pub service_ip_pool_ranges: Vec<IpRange>,
+
+    /// desired total number of deployed Nexus zones
+    // This number probably doesn't make sense alone once we get to a world
+    // where we're deploying multiple _versions_ of Nexus and need to ensure a
+    // minimum number in service at any given time, but for now all we know how
+    // to do is deploy a fixed number of all-the-same-version.
```

Comment:
Sort of?  I would assume in that world this reflects the target number of zones in-service at whatever version is specified in the policy.  So if this were "3", I'd expect us to either deploy 1, remove 1, deploy 1, remove 1, etc. to maintain 3 in service at all times; or else deploy 3, remove 3.  Either way I think this parameter will still make sense.